packages/core/src/common/cluster/request-namespace-list-permissions.test.ts

```diff
@@ -0,0 +1,336 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+
+import type { V1SubjectRulesReviewStatus } from "@kubernetes/client-node";
+import type { DiContainer } from "@ogre-tools/injectable";
+import { getDiForUnitTesting } from "../../main/getDiForUnitTesting";
+import type { RequestNamespaceListPermissionsFor } from "./request-namespace-list-permissions.injectable";
+import requestNamespaceListPermissionsForInjectable from "./request-namespace-list-permissions.injectable";
+
+const createFakeProxyConfig = (statusResponse: Promise<{ body: { status: V1SubjectRulesReviewStatus }}>) => ({
```

Comment:
To consolidate communication in future, this is a `stub` instead of `fake` :)

A `fake` is a *trusted simulation* of something with non-trivial implementation. A `stub` is a trivial substitution of value.

Reply:
Changed.